common/lib/protocol-definitions/src/summary.ts

```diff
@@ -3,8 +3,14 @@
  * Licensed under the MIT License.
  */
 
+/**
+ * TODO
+ */
 export type SummaryObject = ISummaryTree | ISummaryBlob | ISummaryHandle | ISummaryAttachment;
 
+/**
+ * TODO
```

Comment:
```suggestion
 * Type representing the root of the summary tree.
```

Reply:
I failed to notice originally that this is a type overload of an existing type that included docs. I updated it to use inheritDoc